dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Controllers/HealthDataController.cs

```diff
@@ -46,7 +47,7 @@ public class HealthDataController : Controller
         public async Task<ActionResult> Index(Guid personId, Guid? recordId, double weight)
         {
             var item = new Weight(
-                new HealthServiceDateTime(DateTime.UtcNow),
+                new HealthServiceDateTime(new LocalDateTime()),
```

Comment:
A new LocalDateTime is in 1970. You need to use

SystemClock.Instance.GetCurrentInstant().InZone(DateTimeZoneProviders.Tzdb.GetSystemDefault()).LocalDateTime

I'd recommend making an IDateTimeService and injecting it in your controller so you can properly unit test it. You can add a helper method to it to get the current ZoneDateTime or LocalDateTime.